model_analyzer/config/generate/run_config_generator.py

```diff
@@ -88,7 +94,16 @@ def set_last_results(self, measurements):
         ----------
         measurements: List of Measurements from the last run(s)
         """
-        self._curr_pacg.set_last_results(measurements)
+        self._pacg.set_last_results(measurements)
+
+    def _create_run_config(self, model_configs, perf_analyzer_config):
+        if isinstance(model_configs, list):
```

Comment:
yeah it seems like it should be the same in all cases.  Even in the single model case we should move to it being a list with 1 element

Reply:
I discovered that when a model has only one config the recursive function doesn't return it as a list. Rather than try to fix the function, I thought it was easier just to check for this when creating the run config. I'll add a comment about this in the code